package.json

```
@@ -31,33 +31,31 @@
     }
   ],
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=16.6.0"
```

16.6 이상인 특별한 이유가 있나요?

README에는 v16으로 기재되어있어 혼란이 있을 것 같습니다.
```suggestion
    "node": ">=16.0.0"
```

---

```
With undici being used, koreanbots JS/TS SDK v3.1 will only be compatible with nodejs v16.6 and above, because undici only supports above that.
```
Did you read it?

---

Nah